### ISSUE TITLE
FR - solving theme_disclaimer 

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#4303]
## Motivation
THEME_DISCLAIMER is outdated 
source:
```
 $legacy = ['VIEWPORT','THEME_DISCLAIMER', 'IMODE', 'BODYTAG', 'COMMENTLINK', 'OTHERNEWS_LIMIT',
                        'PRE_EXTENDEDSTRING', 'COMMENTOFFSTRING', 'CORE_CSS'];
```
but it is still used in

```
	function sc_theme_disclaimer($parm)
	{
		$pref = e107::getPref();
		return (defined('THEME_DISCLAIMER') && $pref['displaythemeinfo'] ? THEME_DISCLAIMER : '');
	}
```
 

## Proposed Solution
Put this value to theme config (theme prefs) similar way to site disclaimer.

Then:
- theme_config.php will be always in a theme
- there will be always HELP tab to inform users about this option (many of them don't know that you can use the theme pref)
- by default, it could be not editable (it depends on the author)

And because this depends on displaythemeinfo pref, maybe there should be the rule that if theme copyright can't be removed (license terms),  it should be hardcoded in theme HTML files. Or is this intention?  Theme author complies with license terms by using THEME_DISCLAIMER, but with this pref, you let your users very easy to bypass them.

**Comment by CaMer0n** [comment]
@Jimmi08 You're right. It should be hard-coded, and without using a constant and the pref disabled.
